build/cmake/lib/webview_chromium/CMakeLists.txt

```diff
@@ -62,8 +70,149 @@ if(NOT CEF_ROOT)
     if(result)
         message(FATAL_ERROR "Build step for cef failed: ${result}")
     endif()
+endfunction()
+
+if(NOT CEF_ROOT)
+    include(cef_version_info.cmake)
+
+    # Note that CMAKE_OSX_ARCHITECTURES is always defined under macOS, but is
+    # empty if not explicitly specified, so do _not_ use if(DEFINED ...) here.
+    if(CMAKE_OSX_ARCHITECTURES)
+        foreach(osx_arch IN LISTS CMAKE_OSX_ARCHITECTURES)
```

Comment:
I think you can simplify all the foreach a bit to just `foreach(item ${list})`.

Reply:
I've copied this from another wx CMakefile but now I see that we actually use both styles. We probably should standardize on one of them, but I don't know which.